### PR TITLE
Merged the Enemy Unknown from golinski

### PR DESCRIPTION
**Pull request by Anders1232**
I made it based com the github comparation (https://github.com/tracktwo/xcomsave/compare/master...golinski:enemy-unknown), it's compilling, but I haven't tested yet, looking for feedback.
I believe it's working but probably the code I made is not perfectly conformed with the coding style adopted, and I'm open for suggestions where the change can be, and of course anyone can do this too.

### CODE REVIEW

**Comment by Anders1232** [comment]
Ahh, and I also made I little credits area on the usage text, hope I included everyone involved, but u can say if u liked it or wanna any ajustment.

File: json2xcom.cpp [hunk 1]
```diff
@@ -686,6 +686,12 @@ saved_game build_save(const Json& json)
 
 void usage(const char * name)
 {
+	printf("json2xcom, a part of xcomsave, a save game editor for xcom games\n");
```
Comment (tracktwo) [inline json2xcom.cpp]: I'm happy to have the contributors list somewhere (like a contributors file or in the README) but I think this is too long to have in the usage. It'll get way too spammy if there are many more contributors.

File: util.cpp [hunk 2]
```diff
@@ -392,13 +392,21 @@ namespace xcom
         return total;
     }
 
-    std::string build_actor_name(const std::string& package, const std::string& cls, int instance)
+
+	std::string build_actor_name(const std::string& package, const std::string& cls, int instance)
```
Comment (tracktwo) [inline util.cpp]: Looks like some tabs sneaked in. Can you convert them all to 4 spaces to match the existing style please?

File: xcom2json.cpp [hunk 3]
```diff
@@ -566,6 +566,12 @@ void buildJson(const saved_game& save, json_writer& w)
 
 void usage(const char * name)
 {
+    printf("xcom2json, a part of xcomsave, a save game editor for xcom games\n");
```
Comment (tracktwo) [inline xcom2json.cpp]: Same as the other comment

File: xcomwriter.cpp [hunk 4]
```diff
@@ -119,13 +131,21 @@ namespace xcom
 
         virtual void visit(object_property *prop) override
         {
-            if (prop->actor == -1) {
-                io_.write_int(prop->actor);
+            //enemy unbknow
```
Comment (tracktwo) [inline xcomwriter.cpp]: typo :)

**Review by tracktwo — COMMENTED** [review]
Overall looks good, thank you very much for doing this!

File: util.cpp (hunk 2 above)
Comment (Anders1232) [reply util.cpp]: I thought i got rid of all the tabs, 10 minutes.

File: json2xcom.cpp (hunk 1 above)
Comment (Anders1232) [reply json2xcom.cpp]: For now I'll put it in the readme, ok?

File: xcomwriter.cpp (hunk 4 above)
Comment (Anders1232) [reply xcomwriter.cpp]: Thanks for the find =D

**Review by tracktwo — APPROVED** [review]

**Pull request merged**